internal/api/v1beta1/job_types.go

```diff
@@ -45,6 +45,8 @@ type JobStatus struct {
 
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
+// +kubebuilder:resource:scope=Cluster
```

Comment:
I think job scope should be cluster level, like app. Missed this in in the job CRD PR.